mkdocs.yml

```diff
@@ -144,6 +144,12 @@ nav:
           - 'Recovery': variants/protectli_vp46xx/recovery.md
           - 'Hardware Configuration Matrix': variants/protectli_vp46xx/hardware-matrix.md
           - 'Test matrix': variants/protectli_vp46xx/test-matrix.md
+      - 'QEMU Q35 Emulator':
+          - 'Overview': variants/qemu_q35/overview.md
+          - 'Releases': variants/qemu_q35/releases.md
+          - 'Building manual': variants/qemu_q35/building-manual.md
+          - 'Development': variants/qemu_q35/development.md
+          - 'Test matrix': variants/qemu_q35/test-matrix.md
```

Comment:
overview_url: http://docs.dasharo.com/variants/protectli_vp46xx/overview/
        building_url: http://docs.dasharo.com/variants/protectli_vp46xx/building-manual/
        deploy_url: https://docs.dasharo.com/variants/protectli_vp46xx/initial-deployment/
        update_url: http://docs.dasharo.com/variants/protectli_vp46xx/firmware-update/
        recovery_url: http://docs.dasharo.com/variants/protectli_vp46xx/recovery/
        
We also need links to `Hardware Configuration Matrix`, `Initial deployment`, `Recovery` and `Firmware update` for our release automation to be satisfied. These section may contain just one sentence:

QEMU is an emulation environment and not a real hardware thus firmware {deployment, update, recovery} is not applicable.

Although we would probably need a Hardware Configuration Matrix to be filled with standard information about devices emulated by QEMU.